databricks/sdk/mixins/workspace.py

```diff
@@ -0,0 +1,40 @@
+import typing
+
+from ..service import workspace
+
+ExportFormat = workspace.ExportFormat
+Language = workspace.Language
+
+
+def _fqcn(x: any) -> str:
+    return f'{x.__module__}.{x.__name__}'
+
+
+class WorkspaceExt(workspace.WorkspaceAPI):
+
+    def upload(self,
+               path: str,
+               content: typing.BinaryIO,
+               *,
+               format: ExportFormat = ExportFormat.AUTO,
+               language: Language = None):
+        if not isinstance(format, ExportFormat):
+            raise ValueError(
+                f'format is expected to be {_fqcn(ExportFormat)}, but got {_fqcn(format.__class__)}')
+        if language and not isinstance(language, Language):
+            raise ValueError(
+                f'language is expected to be {_fqcn(Language)}, but got {_fqcn(language.__class__)}')
+        data = {'path': path, 'format': format.value}
+        if language:
+            data['language'] = language.value
+        return self._api.do('POST', '/api/2.0/workspace/import', files={'content': content}, data=data)
```

Comment:
The `data` kwarg in requests appears to be for form data, not JSON body.

What is the resulting request content type here?

Reply:
Separate question: does this same API work for uploading regular files? I notice we use the `/workspace-files//import-file` API elsewhere to import regular files.

Reply:
![image](https://github.com/databricks/databricks-sdk-py/assets/259697/c821f913-d617-41b2-9e0c-b66e3a88d500)